functions/Handlers.js

```diff
@@ -20,6 +20,17 @@ class RequestError extends Error {
 }
 
 const COLLECTION_NAME_ORDERS = 'orders';
+const CUSTOMER_CAPABILITIES = ['publish', 'subscribe', 'history', 'presence'];
+const RIDER_CAPABILITIES = ['publish', 'subscribe'];
+
+const checkAblyApiKey = (key) => {
+  if (typeof key !== 'string') {
+    throw new Error('Environment variable for Ably API key not found.');
```

Comment:
I think we should at least have error messages which tell you which env var you need to set

Reply:
@owenpearson has pushed additional commits that refactor and improve this structure, so I'm now going to resolve this conversation on his behalf as I know that he is now happy with this solution.